js/scripts/test.js

```diff
@@ -0,0 +1,21 @@
+const cp = require('child_process');
+
+const mochaBin = require.resolve('mocha/bin/mocha');
+
+if (process.platform !== 'win32' && process.platform !== 'darwin') {
+  execSync('yarn rebuild-slow');
+  spawnSync(process.argv0, [mochaBin, '--expose-gc', 'js/spec/index-slow-spec.js']);
+  execSync('yarn rebuild');
+}
+spawnSync(process.argv0, [mochaBin, '--expose-gc', 'js/spec/index-spec.js']);
```

Comment:
I had to add this script to rebuild and run tests when on Linux, couldn't figure out how to do it from package.json